lib/repl.js

```diff
@@ -1126,7 +1126,7 @@ function isRecoverableError(e, self) {
       self._inTemplateLiteral = true;
       return true;
     }
-    return /^(Unexpected end of input|Unexpected token)/.test(message);
+    return /^(Unexpected end of input|Unexpected token|missing \) after argument list)/.test(message);
```

Comment:
As much as it pains me, the indentation + the regexp itself are at 80 characters, and multiline regexp isn't a thing in js.
It can be made a little shorter by moving the regexp to a variable, or if you'd rather I can cook something up which doesn't use regexps, maybe:

```javascript
var prefixes = [
    'Unexpected end of input',
    // ...
];

return prefixes.some(message.startsWith.bind(message));
```